packages/core/src/data-editor/data-editor.tsx

```diff
@@ -1994,6 +2116,19 @@ const DataEditorImpl: React.ForwardRefRenderFunction<DataEditorRef, DataEditorPr
     const onKeyDown = React.useCallback(
         (event: GridKeyEventArgs) => {
             const fn = async () => {
+                let cancelled = false;
+                if (onKeyDownIn !== undefined) {
+                    onKeyDownIn({
+                        ...event,
+                        cancel: () => {
+                            cancelled = true;
+                            event.cancel();
```

Comment:
need to figure out if this is actually desired, probably not. This also calls `stopPropagation`